tests/test-writing-funct-acq.py

```diff
@@ -0,0 +1,10 @@
+from silverlabnwb import NwbFile
+import os
+
+if __name__ == '__main__':
+    for subdirs in os.walk("C:\\Users\\Alessandro\\Documents\\UCL-projects\\silverlab\\Data\\"):
+        if subdirs[0].endswith('FunctAcq'):
+            import_folder_name = subdirs[0].split("\\")[-1]
+            print("writing "+import_folder_name)
+            with NwbFile("C:\\Users\\Alessandro\\Documents\\UCL-projects\\silverlab\\Data\\by-pysilverlab\\"+import_folder_name+"-by-pysilverlab.nwb", mode='w') as nwb:
+                nwb.import_labview_folder(subdirs[0])
```

Comment:
I assume this is to set up the MATLAB comparison tests? If so, not sure it belongs here, but it might not be a bad idea to have it as an optional test! We do something similar in `test_labview_import.py`, in the `TestFullImporting` class, where we use environment variables to:
- set the data directory, and
- only run the tests if desired